src/status_im/contexts/wallet/send/utils_test.cljs

```diff
@@ -27,3 +28,64 @@
               "0x11" {:status   :pending
                       :id       61
                       :chain-id :420}})))))
+
+(deftest test-network-amounts
+  (testing "Correctly calculates network amounts for transaction with native token"
+    (let [route          [{:amount-in "0xde0b6b3a7640000"
+                           :to        {:chain-id "1"}}
+                          {:amount-in "0xde0b6b3a7640000"
+                           :to        {:chain-id "2"}}]
+          token-decimals 18
+          native-token?  true
+          to?            true
+          result         (utils/network-amounts {:route          route
+                                                 :token-decimals token-decimals
+                                                 :native-token?  native-token?
+                                                 :to?            to?})
+          expected       {"1" (money/bignumber "1")
+                          "2" (money/bignumber "1")}]
+      (doseq [[chain-id exp-value] expected]
+        (is (money/equal-to (get result chain-id) exp-value)))))
+
+  (testing "Correctly calculates network amounts for transaction with non-native token"
+    (let [route          [{:amount-out "0x1e8480"
+                           :from       {:chain-id "1"}}
+                          {:amount-out "0x1e8480"
+                           :from       {:chain-id "2"}}]
+          token-decimals 6
+          native-token?  false
+          to?            false
+          result         (utils/network-amounts {:route          route
+                                                 :token-decimals token-decimals
+                                                 :native-token?  native-token?
+                                                 :to?            to?})
+          expected       {"1" (money/bignumber "2")
+                          "2" (money/bignumber "2")}]
+      (doseq [[chain-id exp-value] expected]
+        (is (money/equal-to (get result chain-id) exp-value))))))
+
+(deftest test-network-values
+  (testing "network-values correctly handles and passes BigNumber values"
```

Comment:
kind of unclear to what the purpose of this function is, it seems to just be an identity function? 🤔

Reply:
I realised this was not needed anymore, so ended up removing this function